README.md

```diff
@@ -73,7 +73,7 @@ path|description
 **Train**
 path|compute|environment|description
 -|-|-|-
-[workflows/train/deepspeed/cifar/job.py](workflows/train/deepspeed/cifar/job.py)|AML - GPU|docker|train CIFAR-10 using DeepSpeed and PyTorch
+[workflows/train/deepspeed/cifar/job.py](workflows/train/deepspeed/cifar/job.py)|AML - GPU|curated|train CIFAR-10 using DeepSpeed and PyTorch
```

Comment:
going to fail `smoke` test - don't manually edit README.md, need to edit logic in `readme.py` (I think as it is this will simply put "unknown" here, which is fine)

Reply:
Sounds good. It did say unknown, which looks weird so I modified it. Shall I revert it to unknown?

Reply:
simply run `python readme.py` - it will also format the code, which is why the smoke test is failing currently (readme check is after)

Reply:
Got it. Done.